web/src/main/java/org/fao/geonet/monitor/link/LinkCheckerDefault.java

```diff
@@ -23,4 +32,8 @@ public boolean check() {
     public String toString() {
         return url;
     }
+
+    public void setLinkTypes(Set linkTypes) {
```

Comment:
Possibly, I can experiment with that.

I'm a newbie with spring and dependency injection. I thought it passes a non-typed `Set<>` but I might be wrong.